src/utils/middlewares/azure_user_attributes_manage.ts

```diff
@@ -86,17 +101,71 @@ export const AzureUserAttributesManageMiddleware = (): IRequestMiddleware<
 
   if (subscriptionId.startsWith("MANAGE-")) {
     // MANAGE Flow
-    const authInfo: IAzureUserAttributesManage = {
-      email: userEmail,
-      kind: "IAzureUserAttributesManage"
-    };
-
-    return E.right<
-      IResponse<
-        "IResponseErrorForbiddenNotAuthorized" | "IResponseErrorInternal"
-      >,
-      IAzureUserAttributesManage
-    >(authInfo);
+    // TODO: The condition MUST be removed after io-function-services update
```

Comment:
Please explain `MUST`. Do you see possible inconsitencies?

Reply:
The "if" condition on authorizedCIDRsModel (and related else) is temporary because we need to deploy this development in order to update `io-functions-services` impacted functions (UpdateService, UpdateServiceLogo, ...).
Because in those functions we have to update AzureUserAttributeManage with new authorizedCIDRs model.
So the temporary behaviour is useful to have consistency in MANAGE flow during the time we need to update `io-functions-services`.

Reply:
I don't understand the need. I only see two possible state for the target function app (`io-functions-services` for the example):
1. it has the **current** version of `io-functions-commons` installed, and **it does not** lookup for cidrs
2. it has the **new** version of `io-functions-commons` installed, and **it does** lookup for cidrs

Do you think we need a third, intermediate state?